### PR TITLE
chore: Add healthcheck endpoint

### PR DESCRIPTION
**Pull request by rhysbarrett**
### JIRA

[Jira Ticket](https://companieshouse.atlassian.net/browse/SR-92) 

### Change description

Implements and tests a very basic healthcheck endpoint (/suppressions/healthcheck), which returns status 200 (OK).

### Swagger Screenshot

swagger

### Work Checklist

- [x] Tests added where applicable
- [x] Test coverage of new code meets target of 80%

### CODE REVIEW

File: src/main/java/uk/gov/companieshouse/controller/HealthcheckController.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+
+@RestController
+@RequestMapping("/suppressions/healthcheck")
```
Comment (geek-soft) [inline src/main/java/uk/gov/companieshouse/controller/HealthcheckController.java:14]: Wasn't `actuator` health check sufficient?

Creating custom health check endpoint seems unnecessary at first.

File: README.md [hunk 2]
```diff
@@ -30,6 +30,7 @@ Development mode is available for this service in [Docker CHS Development](https
 
 ### Health
 `/actuator/health`
```
Comment (geek-soft) [inline README.md:32]: Have you considered changing URL of the actuator health check to be simple `/healthcheck`?

File: README.md [hunk 3]
```diff
@@ -30,6 +30,7 @@ Development mode is available for this service in [Docker CHS Development](https
...
+`/suppressions/healthcheck`
```
Comment (geek-soft) [inline README.md:33]: Spec specifies URL to be `/healthcheck`, does not it?

File: src/main/java/uk/gov/companieshouse/controller/HealthcheckController.java (hunk 1 above)
Comment (rhysbarrett) [reply src/main/java/uk/gov/companieshouse/controller/HealthcheckController.java:14]: I was following the guidelines, which dictate the healthcheck URI. I wasn't aware actuator's endpoint could be changed, but I'll look into it.

File: README.md (hunk 2 above)
Comment (rhysbarrett) [reply README.md:32]: As above, I hadn't, but I'll investigate.

File: README.md (hunk 3 above)
Comment (rhysbarrett) [reply README.md:33]: My bad, I thought `suppressions` was the base of the URL and couldn't be changed.

**Review by geek-soft — APPROVED** [review]
image

**Pull request merged**